PRNTRS Client/main.css

```diff
@@ -1,62 +1,147 @@
-.signin{
-    color:red;
-    font-family: 'Open Sans', sans-serif;
-    font-size: 22px;
-    margin-top: 100px;
-    margin-left: 300px;
-    border-right: solid;
-    border-color: black;
+@import url('https://fonts.googleapis.com/css2?family=Comfortaa&display=swap');
+html, body
+{
+	margin: 0;
+  	padding: 0;
+  	height: 100%;
+  	font-family: Comfortaa, sans-serif;
+  	font-style: normal;
+  	font-weight: normal;
+  	font-size: 23px;
+}
+.signin
+{
+    color:#C74E48;
+    
+    position: absolute;
+	width: 142px;
+	height: 24px;
+	left: 97px;
+	top: 32px;
+	text-decoration: none;
+    line-height: 40px;
+	display: flex;
+	align-items: center;
+	letter-spacing: -0.015em;
 
 }
-.register{
-    font-family: 'Open Sans', sans-serif;
-    font-size: 22px;
-    margin-left: 380px;
-    margin-top:-44px;
-    border-left: solid 1px ;
-    padding-left:6px;
+.register
+{
+    text-decoration: none;
+	line-height: 40px;
+	display: flex;
+	align-items: center;
+	letter-spacing: -0.015em;
 
+	color: #000000;
+	position: absolute;
+	width: 142px;
+	height: 30px;
+	left: 437px;
+	top: 28px;
 }
-.welcome{
-    color:black;
-    font-family: 'Open Sans', sans-serif;
-    font-size: 22px;
-    margin-left:300px;
+.welcome
+{
+    position: absolute;
+	width: 411px;
+	height: 82px;
+	left: 237px;
```

Comment:
Maybe just change the `left` value a bit to make sure the `Welcome Back To` comes directly above `PRNTRS CLIENT`.
Other than that, looks great!